gomint-server/src/main/java/io/gomint/server/entity/Entity.java

```diff
@@ -1105,6 +1105,10 @@ public boolean damage( EntityDamageEvent damageEvent ) {
         return !damageEvent.isCancelled();
     }
 
+    public void interact( io.gomint.entity.EntityPlayer player, Vector clickVector ) {
```

Comment:
This can be declared abstract.

Reply:
yeah it should be abstract

Reply:
no it cant if i make it abstract every entity have to implement this method but not every entitay need that